cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4348,6 +4370,26 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    let x = &Operand::new("x", Float);
+    let a = &Operand::new("a", FloatTo);
+
```

Comment:
Note the original patch used fvpromote_low. This was to be consistent with fvdemote. However there was a desire to rename fvpromote_low to fpromote_low and also git rid of fvdemote (combine with fdemote). Now that fvdemote is back, the naming is inconsistent here but that may be OK.